src/content/docs/tutorial-errors/route-teams.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Assign errors to teams and update error status
+metaDescription: Learn how to manage a high number of errors in your app.   
+---
+
+import apmErrorsAssign from 'images/apm_screenshot-crop_errors-triage.webp'
+
+import apmErrorsStatus from 'images/apm_screenshot-crop_errors-status.webp'
+
+import apmErrorsAssignment from 'images/apm_screenshot-crop_errors-assignment.webp'
+
+
+Now that you understand the full scope of your outage and the error group, you can assign the error and update its status. When you assign errors within New Relic, you can transfer all the information you've gathered to the code owners. Managing your errors inbox eliminates information silos across multiple teams, which expedites the resolution process. 
+
+## Objectives [#objectives]
+
+This tutorial guides you through managing your errors so you can deploy fixes faster:
+
+* Learn to assign errors to the correct teams
+* Update the status of your errors
+
+## Manage your error groups [#manage]
+
+<Steps>
+    <Step>
+
+### Assign the error to the correct team
+        
+From the **Error group summary** page, you can assign the error group to the correct team. 
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmErrorsAssign}
+    />  
+
+Assigning an error to a person or team eliminates possible miscommunications. The information that helped you solve the error is delivered directly to the code owner, allowing them to pick up where you left off. 
+
+The assignment is then delivered to the team via email:
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmErrorsAssignment}
+    />  
+
+    </Step>
+    <Step>
+
+### Mark the status of the error 
+
+Once assigned, you can update the status of an error. 
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmErrorsStatus}
+    />  
+
+This functionality has a few different benefits:
+
+* If an error group is expected, you can mark the error as **Ignored**. Expected errors are known to you and the team — they can be noncritical bugs, or they can be errors associated with the end user (like someone using an incorrect password). 
+* We recommend resolving expected errors as much as you can, however. Ignoring an error group doesn't prevent New Relic from reporting the error in the future, which contributes to your data ingest. 
```

Comment:
`which contributes to your data ingest. ` Is this the real reason to avoid this? Isn't it more like "doesn't fix the root cause"? Unless errors are a huge reason for data ingest this seems off to me.